src/components/experiments/wizard/ExperimentForm.tsx

```diff
@@ -188,6 +192,10 @@ const ExperimentForm = ({
           setActiveStageId(stageId)
           void updateStageState(stages[currentStageIndex])
 
+          errorStages.includes(stageId) &&
+            getStageErrors(stages[stageId]).then((stageErrors) =>
+              formikProps.setTouched(setNestedObjectValues(stageErrors, true)),
+            )
```

Comment:
This doesn't have to be a single statement :slightly_smiling_face: 

```suggestion
          if (errorStages.includes(stageId)) {
            getStageErrors(stages[stageId]).then((stageErrors) =>
              formikProps.setTouched(setNestedObjectValues(stageErrors, true)),
            )
          }
```